apps.yaml

```diff
@@ -149,7 +149,7 @@ appsInfo:
     integration: Httpbin is by default available for developers to use.
   ingress-nginx:
     title: Ingress-NGINX
-    appVersion: 1.1.2
+    appVersion: 1.7.1
```

Comment:
It seems that community put confuses chart version with the app version.
See: https://github.com/kubernetes/ingress-nginx/issues/10190
Here the appVersion should be `4.6.1`